src/Mjolnir.ts

```diff
@@ -241,8 +249,19 @@ export class Mjolnir {
         return this.automaticRedactionReasons;
     }
 
+    /**
+     * Start Mjölnir.
+     */
     public start() {
         return this.client.start().then(async () => {
+            // Start the bot.
+            await this.client.start();
```

Comment:
I don't think this should be here or is a mistake? mjolnir is handling every event twice (found this after seeing it respond twice to `!mjolnir help`) and it doesn't know it has already handled them.

Reply:
Oops, good catch.